.github/workflows/ci.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python-version }}
-      - name: Install setuptools on Python 3.12
+      - name: Install setuptools on Python 3.13
```

Comment:
The setuptools is not part of the python default packages since 3.12. Therefore this should be `Install setuptools on Python >= 3.12`.